MiauAPI/Validators/CreatedAddressRequestValidator.cs

```diff
@@ -32,13 +26,15 @@ public bool IsRequestValid(CreatedAddressRequest request, out IEnumerable<string
         }
 
         // Check reference
-        if (!Validate.IsTextInRange(request.Reference, 100, nameof(request.Reference), out var referenceError))
+        if (request.Reference != null &&
```

Comment:
Não deveria ser `Validate.IsNull(...)`?

Reply:
Fiquei na dúvida se o que você apontou foi porque eu não usei a função `Validate.IsNull(…)` ou se foi porque eu estou conferido se `não é nulo`.
Se foi por causa da função, foi erro meu mesmo, vou corrigir!
Se foi por causa da parte de `não ser nulo`, eu coloquei assim porque `Reference`, `Complement`e `Destinatary` não são campos obrigatórios, então só verifico o tamanho da string se os valores não forem nulos.

Reply:
Ah, faz sentido.
É, só use `Validate.IsNull(…)` se for obrigatório que a propriedade não seja nula.